releases/v0.3.0/RELEASE_NOTES.md

```diff
@@ -0,0 +1,49 @@
+# Cluster API Provider for Cloudstack (CAPC) Release Notes
+
+## Version 0.3.0
+
+These Release Notes are for the customer downloading and deploying CAPC private Version 0.3.0 released on 01/21/2022.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+THE SOFTWARE SHOULD NOT BE USED FOR PRODUCTION WORKLOADS
+
+* This software creates virtual machine instances in a target CloudStack environment and configures them into a Kubernetes cluster.
+* This is a release of partial functionality for mid-development-process review by stakeholders.
+    * Cluster creation, deletion and upgrade with CAPC via EKS Anywhere.
+    * Support for CloudStack Affinity Groups
+    * Baseline Components
+        * Apache CloudStack 4.14
+        * Cluster API clusterctl v0.4.4
+        * Guest OS - RHEL 8
+        * Host OS - CentOS 7
+        * Cilium Container Network
+    * Note: The default mode of operation for the deployed Kubernetes cluster components is to use self-signed certificates.  Options exist for use of an enterprise certificate authority via cert-manager (https://cert-manager.io/docs/configuration/).  Detailed configuration of this component is outside the scope of this release.
+* The following pre-conditions must be met for CreateCluster feature to operate as designed.
```

Comment:
Limitation to _CreateCluster_ is a v0.1.0 artifact.  These pre-conditions apply to all CAPC operations now.